appinventor/appengine/src/com/google/appinventor/shared/rpc/user/Config.java

```diff
@@ -7,6 +7,8 @@
 
 import com.google.gwt.user.client.rpc.IsSerializable;
 import java.io.Serializable;
+import java.util.ArrayList;
```

Comment:
`ArrayList` import does not appear to be needed.

Reply:
Removed